src/test/java/integration/TabsTest.java

```diff
@@ -26,7 +26,7 @@ void setUp() {
   @Test
   @Video
   void userCanBrowseTabs_webdriver_api() {
-    openFile("page_with_tabs.html");
+    setTimeout(1000);
```

Comment:
Isn't `openFile("page_with_tabs.html");` removed by mistake?

Reply:
No, the same line `openFile("page_with_tabs.html");` is executed in `setUp()` method.